pennylane_orquestra/ibmq_device.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The IBMQ device class for PennyLane-Orquestra.
+"""
+import os
+import warnings
+
+from pennylane_orquestra.orquestra_device import OrquestraDevice
+
+
+class QeIBMQDevice(OrquestraDevice):
+    """The Orquestra IBMQ device."""
+
+    short_name = "orquestra.ibmq"
+
+    qe_component = "qe-qiskit"
+    qe_module_name = "qeqiskit.backend"
+    qe_function_name = "QiskitBackend"
+
+    def __init__(self, wires, shots=1024, backend="ibmq_qasm_simulator", **kwargs):
+
+        self._token = os.getenv("IBMQX_TOKEN") or kwargs.get("ibmqx_token", None)
```

Comment:
This would mean that the environment variable would take precedence over the kwarg, right? Shouldn't it be the other way around; if a token is passed as a kwarg, then that one is used (even if there's an environmental var token)?
```suggestion
        self._token = kwargs.get("ibmqx_token", None) or os.getenv("IBMQX_TOKEN")
```

Reply:
Good catch! Will update the Qiskit plugin too :smile: Thanks!